.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
why this change?

Reply:
i don't know ,these changes aren't showing on my workflow